### DISCUSSION TITLE
I didn't receive the admin panel link at the end of the installation. در انتهای فرایند نصب لینک پنل به نمایش در نیامد

### DISCUSSION THREAD

**Discussion by aliidn** [hiddify/Hiddify-Manager#3682] [Q&A سوالات رایج]
سلام وقت به خیر
من پنل رو روی سرور آمازون (EC2) نصب کردم. ولی در انتها لینک پنل رو بهم نشون نداد. چجوری میتونم لینک پنل رو دوباره بگیریم؟؟
یک بار دیگه هم فرایند نصب رو رفتم ولی فرقی نکرد.

![Screenshot (244)](https://github.com/hiddify/Hiddify-Manager/assets/93125266/8cd67f5e-3215-46c1-81f8-a5f8f314a847)

**Comment by af60** [comment]
در انتها دوباره bash <(curl i.hiddify.com/release) رو اجرا کنید و مراحل دنبال کنید. به احتمال زیاد نصب انجام خواهد شد.

**Reply by kerosina** [reply]
This worked for me. Thank you !! It should be in the FAQ
